docs/caddy.md

```diff
@@ -0,0 +1,63 @@
+## How to configure Caddy reverse proxy from port 443 to 1234
+
+### Steps to install Caddy to your ubuntu machine
+
+```
+$ git clone "https://github.com/caddyserver/caddy.git"
+$ cd caddy/cmd/caddy/
+$ go build
+$ sudo mv caddy /usr/bin/
+
+Test that it worked: $ caddy version
+
+$ sudo groupadd --system caddy
+$ sudo useradd --system \
+    --gid caddy \
+    --create-home \
+    --home-dir /var/lib/caddy \
+    --shell /usr/sbin/nologin \
+    --comment "Caddy web server" \
+    caddy
+```
+
+### Create a caddy file
+
+`sudo vim /etc/caddy/Caddyfile`
+
+### Caddy file example
+
+```
+https://domain {
+  @options {
+        method OPTIONS
+  }
+
+  handle @options {
+        header Access-Control-Allow-Origin *
+        header Access-Control-Allow-Headers *
+        header Access-Control-Request-Method *
+        header Access-Control-Allow-Credentials true
+        respond 204
+  }
+
+  log
+  route * {
+   reverse_proxy /* :1234
```

Comment:
@ionutiorgulescu Change this to `:3000` (on this machine, we're proxying 3000 to access the node app, not 1234 which is the Lotus rcp api)

Reply:
@ionutiorgulescu Perfect - thanks!